components/Nav.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+  <nav>
+    <div class="navbar container mx-auto px-4">
+      <ul id="menu_list">
+        <li><a class="text-lg font-bold" href="#about">Despre Alpha</a></li>
+        <li><a class="text-lg font-bold" href="#testimonials">Testimoniale</a></li>
+        <li><a class="text-lg font-bold" href="#register">Încriere</a></li>
+      </ul>
+      <button id="toggle" @click="onOfMenu(toggle = !toggle)">
+        <span class="nav_btn_actv"></span>
+        <span class="nav_btn_actv"></span>
+      </button>
+      <a href="#home"><img id="logo" src="../static/logo.svg" alt=""></a>
+    </div>
+  </nav>
+</template>
+
+<script>
+export default {
+  data () {
+    return {
+      toggle: false
+    }
+  },
+  methods: {
+    onOfMenu (e) {
+      console.log(this.toggle)
+      if (this.toggle === true) {
+        document.getElementById('menu_list').classList.add('active')
+        document.getElementsByClassName('nav_btn_actv')[0].classList.add('on_off_btn')
+        document.getElementsByClassName('nav_btn_actv')[1].classList.add('on_off_btn')
+      } else {
+        document.getElementById('menu_list').classList.remove('active')
+        document.getElementsByClassName('nav_btn_actv')[0].classList.remove('on_off_btn')
+        document.getElementsByClassName('nav_btn_actv')[1].classList.remove('on_off_btn')
+      }
```

Comment:
te rog frumos sa cauti putin despre cum functioneaza class binding in Vue

incepe de aici: https://vuejs.org/v2/guide/class-and-style.html